docs/extensions.rst

```diff
@@ -30,6 +30,7 @@ explicitly invite it (e.g. everything by default is *Private*). These currently
 include (see the docstrings in the specific components for details):
```

Comment:
Done. Per the documentation, the module itself describes what may be used; the list is just a brief outline, so you know where to find stuff.